### PR TITLE
Hello from Taras

### PR DESCRIPTION
**Pull request by taraskaa**
Hello I add the HTML calculator

### CODE REVIEW

File: Calc Project/calc.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<html>
```
Comment (roypeled) [inline Calc Project/calc.html:1]: This is very cool

File: file2.txt [hunk 2]
```diff
@@ -1 +1,2 @@
 new file
+Hello from Taras
```
Comment (roypeled) [inline file2.txt]: Pleas add your last name

**Review by roypeled — CHANGES_REQUESTED** [review]

File: file2.txt (hunk 2 above)
Comment (taraskaa) [reply file2.txt]: Kobelnyk

Comment (taraskaa) [reply file2.txt]: added please

**Comment by taraskaa** [comment]
added

**Pull request merged**